doc/command-line.md

```diff
@@ -755,6 +755,13 @@ Related lingroup subpaths will be grouped in output, but exact ordering may chan
 Optionally, `genome` can instead report classifications at a desired `rank`,
  regardless of match threshold (`--rank` argument, e.g. `--rank species`).
 
+If using `--lins` taxonomy, you can also provide a `--lingroup` file containing two
```

Comment:
Is this warning something that can be generated at the command-line level? If so, should we remove the "use caution" (or modify it to be less forbidding) in favor of having improved UX and warnings built in? Inquiring minds...

Reply:
modified to be less forbidding! 
We detect and warn users if no rank remains for classification, but I don't specifically warn that this might be a result of `--lingroup` and `--rank` together. I could add that if you think it would be more user friendly?

Reply:
note: it could also happen because a users is not expecting LIN positions to be 0-based, so they've entered rank "incorrectly"...